containerd-shim-spin/src/engine.rs

```diff
@@ -223,7 +223,7 @@ impl SpinEngine {
         let mut runtime_config = RuntimeConfig::new(PathBuf::from("/").into());
         // Load in runtime config if one exists at expected location
         if Path::new(RUNTIME_CONFIG_PATH).exists() {
-            runtime_config.merge_config_file(RUNTIME_CONFIG_PATH);
```

Comment:
thought: We chatted about this yesterday, but we could make  `RUNTIME_CONFIG_PATH` an env (defaults to the root) and set an env in the yaml that points to a different location.  not blocking, just an option

Reply:
Yeah I remember this discussion. I can work on this in another PR. (nit: I prefer `SPIN_RUNTIME_CONFIG_PATH`)